<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerControl</class>
 <widget class="QDockWidget" name="PowerControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Power Control</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>VDDH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>I2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="vdd3doubleSpinBox">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="suffix">
       <string> V</string>
      </property>
      <property name="minimum">
       <double>3.000000000000000</double>
      </property>
      <property name="maximum">
       <double>3.600000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>3.300000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>VDD1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>VDD3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>I1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QDoubleSpinBox" name="i1doubleSpinBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> uA</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="i0doubleSpinBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> uA</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>I0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="vddhdoubleSpinBox">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="suffix">
       <string> V</string>
      </property>
      <property name="minimum">
       <double>6.700000000000000</double>
      </property>
      <property name="maximum">
       <double>11.900000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>9.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="i2doubleSpinBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> uA</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="vdd1enButton">
      <property name="text">
       <string>Enable</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="vdd3enButton">
      <property name="text">
       <string>Enable</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="vdd1doubleSpinBox">
      <property name="suffix">
       <string> V</string>
      </property>
      <property name="minimum">
       <double>0.950000000000000</double>
      </property>
      <property name="maximum">
       <double>1.600000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>1.200000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
